RFCs/FS-1126-allow-lower-case-du-cases when-require-qualified-access-is specified.md

```diff
@@ -44,6 +47,26 @@ type DU = a
 type DU = | a
 
 type DU = | a
+
+[<RequireQualifiedAccess>]
+type DU =
+     | a of int
+     | B of string
+     | C
+     | ``D`` of bool
+     | ``d``
+     
+ [<RequireQualifiedAccess>]
+ type DU = ``a``
+
+ [<RequireQualifiedAccess>]
+ type DU = ``A``
+
+ [<RequireQualifiedAccess>]
+ type DU = | ``a``
+
+ [<RequireQualifiedAccess>]
+ type DU = | ``A``
```

Comment:
These are identical to the ones above. What’s the merit, what’s the difference, if any? A comment may help.

Reply:
I think you are right :) . As this is RFC is about lowercase DU I thought adding an example with Uppercase might help . But based on your comment I guess not :)